stacker/hooks/aws_lambda.py

```diff
@@ -468,6 +489,18 @@ def create_template(self):
         provider.region
     )
 
+    # Check if we should walk / follow symlinks
+    followsymlinks = kwargs.get('followsymlinks')
```

Comment:
I'd like it more if `False` was used as a default value, but other non-boolean values were rejected instead of silently ignored. Something like:

```python
follow_symlinks = kwargs.get('follow_symlinks', False)
if not isinstance(follow_symlinks, bool):
    raise ValueError('follow_symlinks option must be a boolean')

# ...
``

Reply:
hmm yeah, I like your idea better. Thanks for the suggestion... updating now.